ThirdParty/SwiftKeychainWrapper/KeychainWrapper.swift

```diff
@@ -16,19 +17,21 @@ let SecClass: String! = kSecClass as String
 let SecAttrService: String! = kSecAttrService as String
 let SecAttrGeneric: String! = kSecAttrGeneric as String
 let SecAttrAccount: String! = kSecAttrAccount as String
+let SecAttrAccessGroup: String! = kSecAttrAccessGroup as String
 
+/// KeychainWrapper is a class to help make Keychain access in Swift more straightforward. It is designed to make accessing the Keychain services more like using NSUserDefaults, which is much more familiar to people.
 public class KeychainWrapper {
-   private struct internalVars {
+    // MARK: Private static Properties
+    private struct internalVars {
         static var serviceName: String = ""
+        static var accessGroup: String = ""
     }
 
     // MARK: Public Properties
 
-    /*!
-    @var serviceName
-    @abstract Used for the kSecAttrService property to uniquely identify this keychain accessor.
-    @discussion Service Name will default to the app's bundle identifier if it can
-    */
+    /// ServiceName is used for the kSecAttrService property to uniquely identify this keychain accessor. If no service name is specified, KeychainWrapper will default to using the bundleIdentifier.
+    ///
+    ///This is a static property and only needs to be set once
```

Comment:
Can you put a space here or does that break some awesome doc feature that I never use?